VideoWeb/VideoWeb/Controllers/VideoEventsController.cs

```diff
@@ -153,14 +152,12 @@ private async Task UpdateConferenceRoomParticipants(Conference conference, Confe
             var roomId = long.Parse(request.ParticipantRoomId);
             var participantId = Guid.Parse(request.ParticipantId);
 
-
             switch (request.EventType)
             {
                 case EventType.Joined:
                     conference.AddParticipantToRoom(roomId, participantId);
                     break;
                 case EventType.Disconnected:
-                    await _consultationResponseTracker.ClearResponses(conference, participantId);
                     conference.RemoveParticipantFromRoom(roomId, participantId);
```

Comment:
you've removed the await here, does the method still need to have `async Task` in the signature?

Reply:
There is still an await at the bottom which updates the conference cache.